lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -393,7 +393,13 @@ def insert_at_position(position)
         end
 
         def update_positions
-          old_position = send("#{position_column}_was") || bottom_position_in_list + 1
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
ActiveRecord version check? Something along those lines?

```ruby
def activerecord_version_at_least?(major, minor = nil)
  ActiveRecord::VERSION::MAJOR >= major && (!minor || ActiveRecord::VERSION::MINOR >= minor)
end

# activerecord_version_at_least?(5, 1)
```

Or is that way of specifying the version a bit awkward and it would be better to go with version strings?

```ruby
def activerecord_version_at_least?(required)
  Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new(required)
end

# activerecord_version_at_least?('5.1')
```